addons/website_slides/static/tests/tours/slides_tour_tools.js

```diff
@@ -28,21 +38,14 @@ var addSection = function (sectionName, backend = false) {
     content: 'eLearning: create section',
     trigger: prefix + 'footer.modal-footer button:contains("Save")',
     run: "click",
-}, {
-	content: 'eLearning: section created empty',
-	trigger: prefix + 'div.o_wslides_slide_list_category_header:contains("' + sectionName + '")',
-    run: "click",
-}];
+}, ...addSectionContent(prefix, sectionName)];
 };
 
 var addVideoToSection = function (sectionName, saveAsDraft, backend = false) {
     const prefix = backend ? ':iframe ' : '';
 	var base_steps = [
+    ...addSectionContent(prefix, sectionName),
```

Comment:
Let's align it with the rest of the array please (otherwise it should even be further out).
```suggestion
...addSectionContent(prefix, sectionName),
```